integrations/tests/test_maltiverse.py

```diff
@@ -0,0 +1,422 @@
+from socket import AF_UNIX, SOCK_DGRAM
+import pytest
+import maltiverse
+import json
+
+from unittest.mock import patch, mock_open
+
+response_example = {
+    "blacklist": [
+        {
+            "description": "Malicious IP",
+            "source": "Threat Intel Source 1",
+            "external_references": [
+                {
+                    "source_name": "mitre-attack",
+                    "external_id": "Some_id",
+                    "url": "some_url",
+                    "description": "something",
+                }
+            ],
+        },
+        {"description": "Suspicious Domain", "source": "Threat Intel Source 2"},
+    ],
+    "type": "ip",
+    "creation_time": "2022-01-01T00:00:00Z",
+    "modification_time": "2022-01-02T00:00:00Z",
+}
+
+
+def assert_expected_schema(response):
+    """Asserts that the response contains the expected schema fields."""
+    list_of_fields = ["blacklist", "type", "creation_time", "modification_time"]
+    for f in list_of_fields:
+        assert f in response
+
+
+def test_get_ip():
+    """Test the `ip_get` method of the Maltiverse class."""
+    example_token = "example_token"
+    example_ip = "77.53.9.158"
+    testing_maltiverse = maltiverse.Maltiverse(auth_token=example_token)
+
+    with patch("maltiverse.requests.get") as mock_get:
+        mock_response = mock_get.return_value
+        mock_response.json.return_value = response_example
+
+        result = testing_maltiverse.ip_get(example_ip)
+        assert_expected_schema(result)
```

Comment:
Check whether the mock is receiving the expected parameters. For instance:
```suggestion
    with patch("maltiverse.requests.get") as mock_get:
        mock_response = mock_get.return_value
        mock_response.json.return_value = response_example

        result = testing_maltiverse.ip_get(example_ip)
        assert_expected_schema(result)
        mock_get.assert_called_once_with(
            'https://api.maltiverse.com/ip/77.53.9.158',
            headers={'Accept': 'application/json', 'Authorization': 'Bearer example_token'}
        )
```